### DISCUSSION TITLE
Search only in English?

### DISCUSSION THREAD

**Discussion by gilipsy** [oleeskild/obsidian-digital-garden#312] [Q&A]
I do two branches, one in English and one in Russian. When I try to use the search, it does not find any Russian words, only English. How do I make it possible to search in Russian? 

Thank you!

**Comment by oleeskild** [comment]
I'm currently working on a new search which should be significantly faster, and support all languages. Hopefully I'll finish it this week. I'll let you know when it's available.

**Reply by gilipsy** [reply]
I'm waiting for my money to come in so I can send you coffee and cookies for your great work ❤️

**Reply by oleeskild** [reply]
I've just released a new version of the template. Try updating it and see if the search works as expected now. Let me know if it works out or not 😄

**Reply by gilipsy** [reply]
I encountered some errors when I was trying to update the template, but it might have been related to a few other experiments I was doing, as well as the fact that I'm a total beginner in coding :) So, I created a new repository and deployed a new website - now I have search functionality in English and Russian :) Thank you!

**Reply by GJRobert** [reply]
Just merged the new template without problems, and love to see non-ASCII in searching!!! Millions of thanks to @oleeskild, coffee on the way.
